shared_model/interfaces/base/signable.hpp

```diff
@@ -48,16 +49,16 @@ namespace shared_model {
 #ifndef DISABLE_BACKWARD
     template <typename Model,
               typename OldModel,
-              typename HashProvider = shared_model::crypto::Sha3_256>
+              typename HashProvider = crypto::DefaultHashProvider>
     class Signable : public Primitive<Model, OldModel> {
 #else
     template <typename Model,
               typename HashProvider = shared_model::crypto::Sha3_256>
     class Signable : public ModelPrimitive<Model> {
 #endif
-     public:
       using HashProviderType = HashProvider;
```

Comment:
is it needed?

Reply:
If somebody passes another HashProvider - yes